bin/check-stale-results.rb

```diff
@@ -60,17 +61,26 @@ def humanize(secs)
     end.compact.reverse.join(' ')
   end
 
+  def get_uri(path)
+    protocol = (settings['api'].key?('protocol') ? settings['api']['protocol'] : 'http')
```

Comment:
I am thinking that as `settings['api]['protocol']` is not standard we should either inspect the host to see if the protocol is embedded or expose it as a cli arg. for the host, user, password it makes sense to re-use those as they are pretty much guaranteed to be there. Matter of preference really but just my 2 cents.

Reply:
I refactored the method `get_uri` to extract the protocol if it's part of the host.